campaign/views.py

```diff
@@ -13,5 +18,25 @@ def get_name_email(request):
         form = forms.ActivistForm(request.POST)
         if form.is_valid():
             activist = form.save()
-            return HttpResponseRedirect('/success/')
+
+            twilio_client = Client(getattr(settings, 'TWILIO_ACCOUNT_SID'),
+                                   getattr(settings, 'TWILIO_AUTH_TOKEN'))
+
+            twilio_client.calls.create(from_=getattr(settings, 'TWILIO_CALLER_ID'),
+                                       to=activist.phone,
+                                       url=request.build_absolute_uri('/outbound/'))
```

Comment:
A reverse urlresolver would probably be better used here https://docs.djangoproject.com/en/1.11/ref/urlresolvers/#reverse